src/java.base/share/classes/sun/nio/ch/NioSocketImpl.java

```diff
@@ -64,8 +64,9 @@
 /**
  * NIO based SocketImpl.
  *
- * This implementation attempts to be compatible with legacy PlainSocketImpl,
- * including behavior and exceptions that are not specified by SocketImpl.
+ * This implementation attempts to be compatible with legacy PlainSocketImpl
+ * (removed in JDK18), including behavior and exceptions that are not specified by
+ * SocketImpl.
```

Comment:
Maybe we should just drop this sentence and not attempt to replace it.

Reply:
Comment removed. See 49125e7